tests/Makefile.am

```diff
@@ -2,8 +2,9 @@ test: test-full
 
 test-prepare:
 	if test -x /sbin/setcap; then \
-	    sudo /sbin/setcap cap_sys_chroot+ep ${sbindir}/iceccd ; \
+	    sudo /sbin/setcap cap_sys_chroot+ep ${builddir}/../daemon/iceccd cap_sys_chroot+ep ${sbindir}/iceccd ; \
```

Comment:
Adding chroot capability to the daemon binary in the builddir seems to resolve the issue where the test hangs in this new test as well as the debug_test.